### ISSUE TITLE
Why do we have shadow classes with single letter for size?

### ISSUE THREAD

**Issue by akhil-gautam** [bigbinary/neeto-ui#1446]
Everywhere for size, it is either full-word(`small, large, etc.`) or 2-letters(`sm, lg, xl`) but for shadows it is just 1 letter(neeto-ui-shadow-xs, neeto-ui-shadow-s, neeto-ui-shadow-l).
It is inconsistent and confusing. 
Can we keep 2 letters for size like `neeto-ui-shadow-sm, neeto-ui-shadow-lg`?

@praveen-murali-ind @amaldinesh7 _A

**Comment by karthiknmenon** [comment]
@praveen-murali-ind _a

**Comment by praveen-murali-ind** [comment]
Yes we can keep 2 letters like `neeto-ui-shadow-sm`, `neeto-ui-shadow-lg`
